core-web/libs/sdk/client/src/lib/utils/page/common-utils.ts

```diff
@@ -1,27 +1,36 @@
 interface PageRequestParamsProps {
     path: string;
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    params: { [key: string]: any } | undefined; // QueryParams are typed as `any` in frameworks
+    params: { [key: string]: any } | undefined | URLSearchParams; // QueryParams are typed as `any` in frameworks
 }
 
-export const getPageRequestParams = ({ path = '', params = {} }: PageRequestParamsProps) => {
+export const getPageRequestParams = ({
```

Comment:
We need to remove this, not in this PR, but we have to.

cc: @rjvelazco @kevindaviladev